airbyte-integrations/connectors/source-salesforce/source_salesforce/streams.py

```diff
@@ -463,6 +463,8 @@ def wait_for_job(self, url: str) -> str:
                     raise AirbyteTracedException(message=message, failure_type=FailureType.config_error, exception=error)
                 else:
                     raise error
+            if job_status != job_info["state"]:
+                self.logger.info(f"Job {self.name}/{job_id} status changed from {job_status} to {job_info["state"]}")
```

Comment:
`job_info["state"]` should probably be `job_info['state']` as it is not allowed to use double quotes within double quotes